### PR TITLE
Add an option to show or hide navigation menu and include it within the group

### PR DESCRIPTION
**Pull request by cntabana**

### CODE REVIEW

**Comment by mackhankins** [comment]
I don't think groups are controlled by gates.  Would be awesome if I could set that in the config.  I've seen it done before with https://github.com/kenepa/resource-lock/blob/58fec965c9b6dd3b322a259b481bfa763abb9e19/config/resource-lock.php#L68

File: config/filament-jobs-monitor.php [hunk 1]
```diff
@@ -2,5 +2,13 @@
...
 return [
+    'navigation_enabled' => true,
 
+    'labels' => [
+        'navigation_group' => 'Settings',
+    ],
+
+    'icons' => [
+        'navigation' => 'heroicon-o-chip',
+    ],
```
Comment (croustibat) [inline config/filament-jobs-monitor.php:13]: I would provide a more readable approach for settings, something like this :

 
```suggestion
    'navigation' => [
        'enabled' => true,
        'group_labels' => 'Settings',
        'icons' => 'heroicon-o-chip',
    ],
```

File: src/Resources/QueueMonitorResource.php [hunk 2]
```diff
@@ -72,6 +70,21 @@ public static function getRelations(): array
         ];
     }
 
+    protected static function getNavigationGroup(): ?string
+    {
+        return config('filament-jobs-monitor.labels.navigation_group');
```
Comment (croustibat) [inline src/Resources/QueueMonitorResource.php:75]: According to new config array :

```suggestion
        return config('filament-jobs-monitor.navigation.group_label');
```

File: src/Resources/QueueMonitorResource.php [hunk 3]
```diff
@@ -72,6 +70,21 @@ public static function getRelations(): array
...
+    }
+
+    protected static function shouldRegisterNavigation(): bool
+    {
+        return (bool) config('filament-jobs-monitor.navigation_enabled');
```
Comment (croustibat) [inline src/Resources/QueueMonitorResource.php:80]:
```suggestion
        return (bool) config('filament-jobs-monitor.navigation.enabled');
```

File: src/Resources/QueueMonitorResource.php [hunk 4]
```diff
@@ -72,6 +70,21 @@ public static function getRelations(): array
...
+    }
+
+    protected static function getNavigationIcon(): string
+    {
+        return config('filament-jobs-monitor.icons.navigation');
```
Comment (croustibat) [inline src/Resources/QueueMonitorResource.php:85]:
```suggestion
        return config('filament-jobs-monitor.navigation.icons');
```

**Review by croustibat — CHANGES_REQUESTED** [review]
hey thank you for this PR, that's great. i would just propose to write config array differently.

**Pull request merged**